test/helpers/mentor_dashboard_helper_test.rb

```diff
@@ -0,0 +1,11 @@
+require "test_helper"
+
+class MentorDashboardHelperTest < ActionView::TestCase
+  test "time_ago_in_words calculates time in days" do
+    assert_equal "2 days", time_ago_in_words(Time.now - 2.days)
+  end
+
+  test "time_ago_in_words calculates time less than one day in hours or minutes" do
+    assert_equal "about 2 hours", time_ago_in_words(Time.now - 2.hours)
+  end
```

Comment:
Let's get some tests around the boundary conditions.

Since the cutoff is `SECONDS_PER_DAY`, it would make sense to have a test for exactly `SECONDS_PER_DAY` ago, `SECONDS_PER_DAY - 1` seconds ago, and `SECONDS_PER_DAY + 1` seconds ago.

Reply:
It looks like Ruby rounds day minus one second up to a day, so I added another test to show that it will display a day minus one minute in hours (the actual cut off is 31 seconds).

Reply:
Neat, I didn't know that.